content/docs/standard/data-package.md

```diff
@@ -12,7 +12,7 @@ sidebar:
   </tr>
   <tr>
     <th>Profile</th>
-    <td><a href="/profiles/2.0/datapackage.json">https://datapackage.org/profiles/2.0/datapackage.json</a></td>
+    <td><a href="/profiles/2.1/datapackage.json">https://datapackage.org/profiles/2.1/datapackage.json</a></td>
```

Comment:
Might be good to make this table a Markdown table (assuming it is supported by the static website generator). That way the profile can just be referenced as:

```
<https://datapackage.org/profiles/2.1/datapackage.json>
```

Guaranteeing there is no discrepancy between the displayed URL and link